packages/edit-site/src/components/sidebar/color-panel.js

```diff
@@ -3,63 +3,54 @@
  */
 import { __experimentalPanelColorGradientSettings as PanelColorGradientSettings } from '@wordpress/block-editor';
 import { __ } from '@wordpress/i18n';
-
-/**
- * Internal dependencies
- */
-import {
-	BACKGROUND_COLOR,
-	GRADIENT_COLOR,
-	LINK_COLOR,
-	TEXT_COLOR,
-} from '../editor/utils';
+import { STYLE_PROPERTY, LINK_COLOR } from '@wordpress/blocks';
 
 export default ( {
 	context: { supports, name },
 	getProperty,
 	setProperty,
 } ) => {
 	if (
-		! supports.includes( TEXT_COLOR ) &&
-		! supports.includes( BACKGROUND_COLOR ) &&
-		! supports.includes( GRADIENT_COLOR ) &&
+		! supports.includes( 'color' ) &&
+		! supports.includes( 'backgrounColor' ) &&
+		! supports.includes( 'background' ) &&
 		! supports.includes( LINK_COLOR )
 	) {
 		return null;
 	}
 
 	const settings = [];
 
-	if ( supports.includes( TEXT_COLOR ) ) {
+	if ( supports.includes( 'color' ) ) {
 		settings.push( {
-			colorValue: getProperty( name, [ 'color', 'text' ] ),
+			colorValue: getProperty( name, STYLE_PROPERTY.color ),
```

Comment:
Personally, this is the major advantage of centralization, and what pushed me to create this PR. Controls don't need to know what's the path of the property they want to access.

Reply:
If the idea is to let control not need to know the shape of the property they want to access, I guess `getProperty` could receive the property instead of the path, and be responsible for resolving the property.

Of course, we may need to change something that does not contain a property e.g: a preset. So I guess in the future we may have a get and setter that receives the property directly and one that receives the paths.

But for now, what we have is ok.

Reply:
Yes! You're so right. At https://github.com/WordPress/gutenberg/pull/25185/commits/6b635adc870a21f56dbabd1710d6302ee467249b and https://github.com/WordPress/gutenberg/pull/25185/commits/7a44eaa8c407fa8c4fdcdf01b45b0e6be43e54df I've renamed the getter & setter to `getStyleProperty` and `setStyleProperty` and they take the property name now. This clean-up makes everything much better :)